install/redis-install.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y apt-transport-https
+$STD apt-get install -y gnupg
+$STD apt-get install -y lsb-release
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Redis"
+curl -fsSL https://packages.redis.io/gpg | sudo gpg --dearmor -o /usr/share/keyrings/redis-archive-keyring.gpg
+echo "deb [signed-by=/usr/share/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/redis.list
```

Comment:
use `$(grep '^VERSION_CODENAME=' /etc/os-release | cut -d'=' -f2)` instead of `$(lsb_release -cs)`

Reply:
either way is fine

Reply:
Ok. Have taken this from https://redis.io/docs/install/install-redis/install-redis-on-linux/